assets/src/styles/blocks/Covers/CoversEditorStyle.scss

```diff
@@ -0,0 +1 @@
+// Empty for now.
```

Comment:
These 2 can be empty because both the old and new version use the same CSS, so it's included in our bundle anyway. Once we removed the old version we can easily move the CSS into here. I added the file because the code splitting is currently set up to require each type of file be present, and I didn't want to get blocked by rewriting it.

Reply:
Shouldn't this change also be applied [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/67d611e28d82545caa21864a14a8cdb61725bb28/classes/blocks/class-covers.php#L83)? 🤔

Reply:
Right, it puts the wrong thing there :+1:

Reply:
We can already make this use the final name, the previous version didn't use `data-render`, so it won't collide. So I updated this line instead https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/564/files#diff-f4f3a78077b528d85adf10a5c03acfa82930b2de5f93086b8d375181b083c9adR3